.github/workflows/test_all_notebooks.yml

```diff
@@ -27,12 +37,13 @@ jobs:
 
       - name: Install dependencies
         run: |
-          sudo apt-get install build-essential
-          pip install --upgrade pip
-          pip install -r requirements/dev.txt
-          pip install -e .[prophet]
-          pip install -e .[torch]
-          pip freeze
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements/dev.txt
+          if [ "$RUNNER_OS" != "Windows" ]; then  # Skip Prophet notebooks on windows for now
+            python -m pip install -e .[prophet]
+          fi
```

Comment:
I think this won't be as straightforward as the Prophet notebook won't be on the exclusion list, hence will result in a fail.

Reply:
I guess we can use the `PROPHET_INSTALLED` constant.

Reply:
Ah I did this in `testing/test_notebooks.py`

```python
if PROPHET_INSTALLED:
    EXCLUDE_NOTEBOOKS.add('od_prophet_weather.ipynb')  # Exclude if fbprophet not installed i.e. on Windows
```

Reply:
Alright, I need to learn to read to the end of the changes :)